packages/xrpl/src/Wallet/index.ts

```diff
@@ -112,7 +116,8 @@ class Wallet {
     privateKey: string,
     opts: {
       masterAddress?: string
-      seed?: string
+      seed?: string,
+      secretNumbers?: string
```

Comment:
Is secretNumbers needed here? (I don't see it used anywhere in the constructor)

* If it is needed, could you add `@param opts.secretNumbers` + a description to the documentation comment?

Reply:
indeed not needed. removed